### PR TITLE
VTCCA-3480 Claim Property journey change to adopt Welsh properties ba…

### PR DESCRIPTION
**Pull request by reddy-yattapu**
…sed Earliest StartDate provided API

### CODE REVIEW

File: app/controllers/propertyLinking/ChooseEvidenceController.scala [hunk 1]
```diff
@@ -51,18 +56,41 @@ class ChooseEvidenceController @Inject()(
 
   def show: Action[AnyContent] = authenticatedAction.andThen(withLinkingSession).async { implicit request =>
     logger.debug("show choose evidence page")
-    businessRatesAttachmentService
-      .persistSessionData(request.ses, UploadEvidenceData.empty)
-      .map(_ => Ok(chooseEvidenceView(ChooseEvidence.form, request.ses.clientDetails)))
+    for {
+      startDate <- propertyLinkingService.findEarliestStartDate(request.ses.uarn)
```
Comment (habeeb01) [inline app/controllers/propertyLinking/ChooseEvidenceController.scala:60]: where are you using this "startDate" ?

Comment (reddy-yattapu) [reply app/controllers/propertyLinking/ChooseEvidenceController.scala:60]: We are using to make decision of backlink redirecting correct page

**Comment by habeeb01** [comment]
LGTM 👍

**Comment by darrell-plant** [comment]
LGTDz 👍

**Pull request merged**